sorc/ncep_post.fd/SURFCE.f

```diff
@@ -2621,12 +2641,8 @@ SUBROUTINE SURFCE
          if(grib=='grib2') then
            cfld=cfld+1
            fld_info(cfld)%ifld=IAVBLFLD(IGET(422))
-           if (ifhr==0) then
```

Comment:
@KarinaAsmar-NOAA Please keep the original setting for modelname 'FV3R' which is hard-wired for 1-hourly max, and new changes for flexible bucket.

Reply:
@KarinaAsmar-NOAA The original setting should also be applied to modelname 'RAPR'.

Reply:
It should be done now.